lib/datastore/request.js

```diff
@@ -196,6 +196,76 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  *   data: {
  *     rating: '10'
  *   }
+ * }, function(err) {
+ *   console.log(key.path); // [ 'Company', 5669468231434240 ]
+ *   console.log(key.namespace); // undefined
+ * });
+ *
+ * //-
+ * // Save a single entity using a provided name instead of auto-generated ID.
+ * //
+ * // Here we are providing a key with name instead of an id. After saving, the
```

Comment:
I changed to ID. I think I associated `name` and `id` in that sentence with a property like `key.name` and `key.id` which are naturally lowercase. Otherwise I like uppercase ID. I changed here to ID to remain consistent.